### DISCUSSION TITLE
Which country has the most 🏰🏰 per square mile?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1024] [Galaxy Trivia] [answered]
**Bonus:** What is the world's largest castle and where is it?

**Comment by peterbaccaro** [comment, marked as answer]
While Germany may have the most amount of castles, **Wales** is the country with the most castles per square mile. Despite its small size, Wales had over 600 castles. Today over 400 castles still stand and are waiting to be explored by visitors.

The world's largest castle by land area is **Malbork Castle**, also known as the Castle of the Teutonic Order in **Malbork, Poland**. Malbork Castle is a medieval fortress located in the town of Malbork in northern Poland. It was built by the Teutonic Knights, a German medieval crusading order, in the 13th century. The castle covers an impressive area of about 143,591 square meters (1,543,705 square feet). It is a UNESCO World Heritage Site and a popular tourist destination in Poland, known for its well-preserved Gothic architecture and historical significance.
